src/app/app.module.ts

```diff
@@ -104,13 +106,22 @@ const Services = [
                         headers: new HttpHeaders().set("Authorization", `Bearer ${accessToken}`),
                     });
                 }
+                return forward(operation);
+            });
 
+            const globalLoaderMiddleware: ApolloLink = new ApolloLink((operation: Operation, forward: NextLink) => {
+                const context = operation.getContext();
+                const skipLoading = Boolean(context.skipLoading);
+                const headers = context.headers as HttpHeaders;
```

Comment:
Might be undefined, you need to provide a default array

Reply:
Fixed